packages/react-form-state/docs/validators.md

```diff
@@ -229,3 +229,11 @@ import {validateList} from '@shopify/react-form-state';
     });
   }}
 ```
+
+### `lastSubmitErrors`
+
+When the user attempts to submit an invalid form with `validateOnSubmit` set to `true`, the validators will run on the form, preventing a call to the `onSubmit` callback if an error arises.
```

Comment:
If this only ever contains client errors, should we maybe add the word `client` errors the name?